packages/web/components/popover.tsx

```diff
@@ -0,0 +1 @@
+export { Popover } from "@headlessui/react";
```

Comment:
What's the purpose of this file?

Reply:
Most likely, we'll extend the popover functionality in the future by adding animations. So I think it's worth using our own import to not have to replace all imports.